src/controllers/profiles.py

```diff
@@ -0,0 +1,53 @@
+from mongoengine.errors import OperationError, DoesNotExist
+
+# Import src.models.properties as prop
+import src.models.profiles as prof
+from ..utils.enums import ControllerStatus
+from ..utils.auth import auth
+
+# Get the user profile
+# I need help on knowing how to bring the id by clicking
+# A "property" so it can be taken there from it xD
+
+
+def read_prof(user_id) -> ControllerStatus:
+    user_id = auth.current_user
+    try:
+        # prop_owner = prop.PropertyDoc.objects.get(id=data["id"])
+        prof_id = prof.ProfileDoc.objects.get()
+    except DoesNotExist:
+        return ControllerStatus.DOES_NOT_EXISTS
+    except OperationError:
+        return ControllerStatus.ERROR
+
+    # Here instead of current, goes owner and uncomment prop owner above
+    if str(user_id["id"]) == str(prof_id["user"].id):
```

Comment:
Convertir un string a un string es un poco redundante
```suggestion
    if user_id["id"] == str(prof_id["user"].id):
```

Reply:
Pensaba que estaba como definiendolo, así como me habías dicho lol

Reply:
No, definir los tipos se hace en los parámetros